app/views/device_stories/_results.html.erb

```diff
@@ -0,0 +1,20 @@
+<table class="table table-hover">
+  <thead>
+    <%= table_sort_header(:device, :device_urn) %>
+    <%= table_sort_header(:device, :custodian_name) %>
+    <%= table_sort_header(:device, :last_values) %>
+    <%= table_sort_header(:device, :last_location_name) %>
+    <%= table_sort_header(:device, :last_seen) %>
+  </thead>
+  <tbody>
+  <% device_stories.each do |device_story| %>
+    <tr onclick = "javascript: location.href = '<%=device_story_path(device_story) %>'">
```

Comment:
We have an existing pattern for this using bootstrap rowlink https://github.com/Safecast/safecastapi/blob/5b7d6595e81d2a8c546d7cb66c1a14b70f43ac82/app/views/bgeigie_imports/index.html.erb#L52

Would be good to do the same here.

This also has me wondering if we _really_ want that.

I find myself sometimes navigating when I wanted to just copy/paste some data from the site.

https://www.robertcooper.me/table-row-links#should-table-rows-be-links-in-the-first-place does a good job calling out some of the downsides.

Wonder if @Azby or @nokton opinions on if we should keep all table rows linkable or not.